modules/weko-schema-ui/weko_schema_ui/schema.py

```diff
@@ -625,17 +625,20 @@ def clean_none_value(dct):
                             if not ddi_schema:
                                 if len(lst_none_idx) > 0:
                                     # delete all None element in @value
-                                    for i in lst_none_idx:
-                                        del node_val[0][i]
+                                    lst_val_idx = list(set(range(len(node_val[0]))) - set(lst_none_idx))
+                                    node_val[0] = [val for idx, val in enumerate(node_val[0])
+                                                   if idx in lst_val_idx]
                                     # delete all None element in all @attributes
                                     for key, val in v.get(self._atr,
                                                           {}).items():
-                                        for i in lst_none_idx:
-                                            del val[0][i]
+                                        temp_val = list(set(range(len(node_val[0]))) - set(lst_none_idx))
```

Comment:
maybe temp_val unnecessary because node_val[0] had delete "none" element

Reply:
Yes, I should use `lst_val_idx ` above instead of `temp_val `